common/ideas/zzz_BOI_ideas.txt

```diff
@@ -1,4 +1,32 @@
 ideas = {
+
+	industrial_concern = {
+		BOI_griffonstone_national_univercity = {
+			allowed = {
+				OR = {
+					original_tag = BOI
+					original_tag = GRI
+					original_tag = GRW
+					original_tag = JES
+					original_tag = WNG
+				}
+			}
+			available = {
+				owns_state = 389
+			}
+			visible = {
+				owns_state = 389
+			}
```

Comment:
just `available`, we want players to know what they need to do if they want this concern